src/plugins/vis_types/pie/public/to_ast.ts

```diff
@@ -70,7 +70,7 @@ export const toExpressionAst: VisToExpressionAst<PartitionVisParams> = async (vi
     emptySizeRatio: vis.params.emptySizeRatio,
     palette: preparePalette(vis.params.palette),
     labels: prepareLabels(vis.params.labels),
-    metric: schemas.metric.map(prepareDimension),
```

Comment:
It looks like this used to sometimes result in _multiple_ accessors being passed as the `metric` arg (which was singular).

It looks like this (accidentally?) resulted in only the final one making it to the renderer function `pie_vis_function.ts` as `args.metric`. I have preserved this behavior since I assume that it is now intended.

cc @stratoula in case she remembers anything about this